docs/operator-guide/upgrade-edp-3.8.md

```diff
@@ -0,0 +1,174 @@
+# Upgrade EDP v3.7 to 3.8
+
+!!! Important
+    We suggest backing up the EDP environment before starting the upgrade procedure.
+
+This section provides detailed instructions for upgrading the EPAM Delivery Platform to version 3.8.1. Follow the steps and requirements outlined below:
+
+
+1. Delete the following resources:
+
+    ```bash
+    kubectl -n edp delete ingress edp-headlamp
+    kubectl -n edp delete ingress edp-tekton-dashboard
+    kubectl -n edp delete ingress el-github-listener
+    kubectl -n edp delete EDPComponents sonar
+    kubectl -n edp delete EDPComponents nexus
+    kubectl -n edp delete EDPComponents argocd
+    kubectl -n edp delete EDPComponents main-keycloak
+    kubectl -n edp delete EDPComponents tekton
+    kubectl -n edp delete EDPComponents github
+    ```
+
```

Comment:
please add description that we remove CustomResourceDefinition edpcomponents.v1.edp.epam.com support and if user use CR in this CRD, before remove CRD user need to remove CR.